apps/ledger-live-desktop/src/renderer/families/cosmos/Delegation/index.tsx

```diff
@@ -264,6 +265,11 @@ const Delegation = ({ account }: { account: CosmosAccount }) => {
 const Delegations = ({ account }: { account: CosmosAccount | TokenAccount }) => {
   if (account.type !== "Account") return null;
 
+  const coinConfig = getCurrencyConfiguration(account.currency);
+  if ("disableDelegation" in coinConfig && coinConfig.disableDelegation === true) {
+    return null;
+  }
+
   return <Delegation account={account} />;
```

Comment:
You need to hide unbondings aswell

Reply:
Ideally we should have added react tests to check behavior with boolean at true / false

Reply:
My bad!